src/modules/product-category/product-category.controller.ts

```diff
@@ -0,0 +1,28 @@
+
+import { Controller, Get, Post, Body, Param, Patch, Delete, UseGuards } from '@nestjs/common';
+import { ProductCategoryService } from './product-category.service';
+import { CreateProductCategoryDto } from './dto/create-product-category.dto';
+import { UpdateProductCategoryDto } from './dto/update-product-category.dto';
+import { ApiTags, ApiBody, ApiParam } from '@nestjs/swagger';
+import { AuthGuard } from 'src/guards/auth.guard';
+import { skipAuth } from 'src/helpers/skipAuth';
+@ApiTags('products/categories')
+@UseGuards(AuthGuard)
+@skipAuth()
+@Controller('products/categories')
```

Comment:
Remove the @skipAuth()  and @UseGuards() decorators

Reply:
Done

Reply:
I have effected the changes